src/utils/graph.ts

```diff
@@ -83,7 +83,7 @@ export type OrderDirection = 'asc' | 'desc'
 
 export type WhereConfig<E extends EntityKey> = {
   key: EntityKeys<E>
-  value: string | number | boolean | string[]
+  value: string | number | boolean | string[] | number[]
```

Comment:
expanded type definition to permit filtering by a string of numbers (archived project IDs). current `WhereConfig` functionality is still compatible without any changes